analytics-api-core/src/main/scala/org/ekstep/analytics/api/util/CacheUtil.scala

```diff
@@ -29,12 +26,32 @@ case class LanguageResponse(id: String, ver: String, ts: String, params: Params,
 // TODO: Need to refactor this file. Reduce case classes, combine objects. Proper error handling.
 
 @Singleton
-class CacheUtil @Inject()(postgresDB: PostgresDBUtil) {
+class CacheUtil @Inject()(postgresDB: PostgresDBUtil, restUtil: APIRestUtil) {
 
   implicit val className = "org.ekstep.analytics.api.util.CacheUtil"
 
   private var cacheTimestamp: Long = 0L;
   private val consumerChannelTable: Table[String, String, Integer] = HashBasedTable.create();
+  private var superAdminChannel: String = "";
+
+  def init()(implicit config: Config) {
+    initDeviceLocationCache()
+    initConsumerChannelCache()
+    initSuperAdminChannelCache()
+  }
+
+  def initSuperAdminChannelCache()(implicit config: Config) {
+    APILogger.log("Updating super admin channel cache ")
+    // get MHRD tenant id using org search API
+    val orgSearchApiUrl = config.getString("org.search.url")
+    val requestBody = """{"request":{"filters":{"channel":"mhrd"},"offset":0,"limit":1000,"fields":["id"]}}"""
```

Comment:
{"channel":"mhrd"} - "mhrd" value should config from config as it is diksha specific

Reply:
Done - added channel as config and defaulted to sunbird